### ISSUE TITLE
Fix header text floating left on mobile devices

### ISSUE THREAD

**Issue by AjayBhatta16** [AjayBhatta16/grabify-clone#19]
Make sure that the h1 and h3 text remains centered on all pages, particularly /dashboard and /viewlink.

**Comment by Harshalyadav** [comment]
May I take this issue?

**Comment by AjayBhatta16** [comment]
> May I take this issue?

Yeah, that's fine. Just be sure to make another branch so you aren't pushing directly to the main.

**Comment by Harshalyadav** [comment]
okay

**Comment by Harshalyadav** [comment]
Could you give me more information about these issues?
